tests/scripts/task_cpp_unittest.sh

```diff
@@ -40,10 +39,20 @@ if grep crttest build/Makefile > /dev/null; then
     make crttest  # NOTE: don't parallelize, due to issue with build deps.
 fi
 
-cd build && ctest --gtest_death_test_style=threadsafe && cd ..
+if grep crttest build/build.ninja > /dev/null; then
```

Comment:
i think it would be better to configure expectations here from e.g. CI_RUN_CRTTEST or a command-line arg to this script, otherwise we could silently fail if someone removes crttest as a target